### PR TITLE
Docker file fix

### PR DESCRIPTION
**Pull request by jrhender**
DockerFile: Now that using npm package, don't need local dist. Also change in path because now in iam-client-example repo
deploy.sh: add script so that can be run from server to deploy express example (haven't add to travis yet to run)

### CODE REVIEW

**Comment by jrhender** [comment]
@dwojno So express example is now updated

File: server/express/Dockerfile [hunk 1]
```diff
@@ -15,26 +11,17 @@ ENV NODE_ENV production
...
 
 # Install PM2
 RUN npm i -g pm2
 
 # Copy package.json, package-lock.json and process.yml
-COPY --chown=node:node examples/server/express/. ./
+COPY --chown=node:node . ./
 
 # Switch to user node
 USER node
```
Comment (dwojno) [inline server/express/Dockerfile:27]: I would make it a really simple one like this, cause this example not have any dev dependencies: 
```
ARG PORT=3333

FROM node:14-alpine

# Set node environment to production
ENV NODE_ENV production

WORKDIR /app

# Update the system
RUN apk --no-cache -U upgrade && apk add python make g++

COPY . .

# install only prod dependencies
RUN npm i --only=production

# Install PM2
RUN npm i -g pm2

# Switch to user node
USER node

# Open desired port
EXPOSE ${PORT}

# Use PM2 to run the application as stated in config file
ENTRYPOINT ["pm2-runtime", "./process.yml"] 
```

**Review by dwojno — COMMENTED** [review]
Maybe we should add a .dockerignore file, with node_modules folder?

File: server/express/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+git pull
+sudo docker build . -t passport-did-auth-express:latest
+sudo docker-compose -f docker-compose.prod.yml up -d
```
[the remark this answers is not in the document]
Comment (jrhender) [reply server/express/deploy.sh:5]: Hmm, I don't think that is necessary. This screenshot shows the last few lines of output from `deploy.sh`. After the docker build complete, I think docker-compose checks if the image with a given tag has changed and recreates the container if necessary.
image

File: server/express/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+git pull
+sudo docker build . -t passport-did-auth-express:latest
```
[the remark this answers is not in the document]
Comment (jrhender) [reply server/express/deploy.sh:4]: done 👍

**Pull request merged**